components/ResourceIDs.tsx

```diff
@@ -1,6 +1,7 @@
 import { fhirJson } from "@fhir-typescript/r4-core";
 import { Button } from "@mantine/core";
 import Link from "next/link";
+export const textGray = "#4a4f4f";
```

Comment:
Looks like this is exported in multiple places (here and _app.tsx) I would add a styles folder with an appColors.tsx file which defines all the custom colors used in the app as consts. Then all files can import from that file :)